CMakeLists.txt

```diff
@@ -303,6 +303,12 @@ ENDIF()
 # exceptions. This is done here:
 SET(ASPECT_USE_FP_EXCEPTIONS ON CACHE BOOL "If ON, floating point exception are raised in debug mode.")
 
+# Clang 6.0 throws random floating point exceptions, which we could not
+# track down. Disable the exceptions for now.
+IF("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang" AND CMAKE_CXX_COMPILER_VERSION VERSION_EQUAL 6.0)
```

Comment:
I think it makes sense to blacklist >=6.0, otherwise people will run into this issue with 6.1, 7.0, etc.. I doubt that this is a bug that will be "fixed".

Reply:
I think there is little harm in doing as you suggest, but they better do fix this -- it's clearly a bug to speculatively execute a statement that might trigger an exception!

(Out of curiosity, which continent are you on right now?)

Reply:
> but they better do fix this

I doubt this will be addressed, so I prefer to be on the safe side on this issue (better to not have FP exceptions than code that doesn't work).

> (Out of curiosity, which continent are you on right now?)

Europe for the next two weeks. :-)